src/content/developers/docs/data-structures/ssz/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Simple Serialize
+description: Explanation of Ethereum's SSZ format.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+Simple serialize (SSZ) is the serialization method used on the Beacon Chain. It replaces the RLP serialization used on the execution layer everywhere across the consensus layer except the peer discovery protocol. SSZ is specifically designed to be deterministic and compatible with Merkleization.
+
+## How does SSZ work? {#how-does-ssz-work}
+
+### Serialization
+
+Ultimately the goal of SSZ serialization is to represent objects of arbitrary complexity as strings of bytes. Each bytestring has a fixed length (32 bytes) called a chunk. These chunks directly become leaves in the Merkle tree representing the object.
```

Comment:
> Ultimately the goal of SSZ serialization

Is it incorrect to just say 'the goal of SSZ'?

Reply:
nah that is much better. Updating now.